package.json

```diff
@@ -46,7 +46,7 @@
     "dev": "next dev",
     "build": "next build",
     "lint": "next lint",
-    "format": "prettier --write **/*.{ts,tsx}",
+    "format": "prettier --write '**/*.{ts,tsx}'",
```

Comment:
What does this do differently?

Reply:
I have no idea, but I think yarn was expanding that glob and causing some problems.

The change is reverted now.